packages/overlay/test/restore-focus.test.js

```diff
@@ -160,3 +160,96 @@ describe('restore focus', () => {
     });
   });
 });
+
+// TODO: temporarily placed in a separate suite using native popover
+// DOM structure, where the content root is slotted into the overlay
+describe('custom content root', () => {
+  customElements.define(
+    'custom-overlay',
+    class extends Overlay {
+      get _contentRoot() {
+        return this.owner;
```

Comment:
Using `owner` here isn't needed, could be also something like `__contentRoot` etc.